Gemfile

```
@@ -25,7 +17,7 @@ gem 'rubocop-rspec', '~> 3.0.0'
 gem 'simplecov', '~> 0.20'
 gem 'stackprof', platform: :mri
 gem 'test-queue'
-gem 'yard', '~> 0.9'
+gem 'yard', '~> 0.9', '>= 0.9.37'
```

Can you tweak to the following?

```suggestion
gem 'yard', '>= 0.9.37', '< 1.0'
```

---

The above suggestion might have been a bit too complex.

Rather, it would be better to keep it as is since it will likely be resolved during `bundle update`. This configuration shouldn't cause any issues and is simple enough.

```suggestion
gem 'yard', '~> 0.9'
```

This is because issues with updating YARD are rare.

---

I changed this to ensure local copies are updated but I guess it doesn't terribly matter